xml/geo_booth_i.xml

```diff
@@ -64,6 +64,75 @@
    parties.
   </para>
  </sect1>
+
+ <sect1 xml:id="sec.ha.geo.booth.ticket-types">
+  <title>Automatic versus Manual Tickets</title>
+  <para>
+   A ticket grants the right to run certain resources on a specific
+   cluster site. Two types of tickets are supported:</para>
+  <itemizedlist>
+   <listitem>
+    <para>Automatic tickets are controlled by the &boothd; daemon.</para>
+   </listitem>
+   <listitem>
+    <para>Manual tickets are managed by the cluster administrator only.</para>
+   </listitem>
+  </itemizedlist>
+
+  <para>Automatic and manual tickets have the following properties:</para>
+  <itemizedlist>
+   <listitem>
+    <formalpara>
+     <title>Automatic and manual tickets can be defined together</title>
+     <para>You can define and use both automatic and manual tickets within the
+      same &geo; cluster.</para>
+    </formalpara>
+   </listitem>
+   <listitem>
+    <formalpara>
+     <title>Manual ticket management remains manual</title>
+     <para>The automatic ticket management is not applied to manually controlled
+      tickets.
+      Manual tickets do not require any quorum elections, cannot fail over
+      automatically, and do not have an expiry time.</para>
+    </formalpara>
+   </listitem>
+   <listitem>
+    <formalpara>
+     <title>Manual tickets are not moved</title>
```

Comment:
Perhaps something like "cannot be moved" or "will not be automatically moved" instead of "are not moved" ?